packages/db/src/translate/providers.ts

```diff
@@ -16,40 +14,33 @@ export const encryptValueWithKMS = (
 /**
 * Translates API Provider record to Postgres Provider record
 *
-* @param {ApiProvider} data - ApiProvider record to translate
+* @param {ApiProvider} record - ApiProvider record to translate
 * @param {Function} [encryptMethod] - The encryption method to use, defaults to encryptValueWithKMS
 * @returns {Promise<PostgresProvider>} Returns a PostgresProvider object
 */
 export const translateApiProviderToPostgresProvider = async (
-  data: ApiProvider,
+  record: ApiProvider,
   encryptMethod: Function = encryptValueWithKMS
 ): Promise<PostgresProvider> => {
   let username: string | undefined;
   let password: string | undefined;
-  if (data.username) {
-    username = await encryptMethod(data.username);
+  if (record.username) {
+    username = await encryptMethod(record.username);
   }
-  if (data.password) {
-    password = await encryptMethod(data.password);
+  if (record.password) {
+    password = await encryptMethod(record.password);
   }
   return ({ // TODO - rewrite this using snake
```

Comment:
Nit: I kind of want to delete this TODO since I prefer to not use something like `snake-camel` and keep our property mappings explicitly defined, but opinions ™️

Reply:
I'm fine with this if @nemreid  is.    If not, we should make a ticket and discuss there.

Reply:
Absent comment from @nemreid, removed: ed0cc78b5